.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+jobs:
+  build:
+    working_directory: /tmp/src
+    docker:
+      - image: appsilon/ci-base:1.0
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - deps1-{{ .Branch }}
+            - deps1-
+      - run:
+          command: |
+            cd /tmp/src/
+            ls
+            cd expfactory.attentionnetworktest
+            R -e 'devtools::install_deps(dependencies = TRUE)'
+      - save_cache:
+          key: deps1-{{ .Branch }}
+          paths:
+            - "/usr/local/lib/R/site-library"
+      - run:
+          command: |
+            echo "Installing package expfactory"
+            R -e "devtools::install_github('expfactory/expfactoryr')"
+            cd /tmp/src/expfactory.attentionnetworktest
+            echo "Running devtools checks and tests for expfactory.attentionnetworktest"
+            R -e 'devtools::check()'
+            R -e 'devtools::test()'
+      - store_artifacts:
```

Comment:
After these tests go through, I'll add in code here that will build a container to deploy, and then push here --> https://hub.docker.com/r/expfactory/attention-network-task/ (we have a Docker Hub organization now, so the docker URIs will be like `expfactory/<experiment>` instead of `vanessa/expfactory-<experiment>`. Just curious - would we would want the expfactoryr package to deploy to cran? That should happen in that repo, of course, although I don't know how to do this proper :)

Reply:
I would hold off deploying to CRAN until it's had some (ab)use.  BTW, in the mass rename the package is `expfactory` rather than `expfactoryr`.